src/main/java/no/nav/pto/veilarbportefolje/util/DateUtils.java

```diff
@@ -48,14 +48,24 @@ public static ZonedDateTime toZonedDateTime(Date date) {
         if (date == null) {
             return null;
         }
-        return ZonedDateTime.ofInstant(date.toInstant(), ZoneId.of("UTC"));
+        return ZonedDateTime.ofInstant(date.toInstant(), ZoneId.of("Europe/Oslo"));
```

Comment:
This can be tricky during summer/winter clock changes. UTC is always the same so its better in that sense.